DigitalOffice/Services/UserService/Validators/GetUserByEmailValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using FluentValidation;
+
+namespace LT.DigitalOffice.UserService.Validators
+{
+    public class GetUserByEmailValidator : AbstractValidator<string>
+    {
+        public GetUserByEmailValidator()
+        {
+            RuleFor(email => email)
+                .NotEmpty()
+                .MaximumLength(129)
+                .WithMessage("Email is too long.")
```

Comment:
This is validated when user is created

Reply:
Yes. So what? :)

Reply:
I'm validating a requested email not the user's one